src/combineReducers.js

```diff
@@ -74,6 +74,46 @@ function assertReducerSanity(reducers) {
   })
 }
 
+function getReducers(config) {
+  var reducers = {}
+
+  Object.keys(config).map(key => {
+    if (typeof config[key] === 'function') {
+      reducers[key] = config[key]
+    }
+  })
+
+  return reducers
+}
+
+function checkStateShape(keys, reducers, state, action) {
```

Comment:
@gaearon what do you think about the `map` here. Is better change to a `for` loop?